DOCS/man/options.rst

```diff
@@ -4212,9 +4212,10 @@ Demuxer
     all. The default is ``auto``, which behaves like ``recursive`` with
     ``--shuffle``, and like ``lazy`` otherwise.
 
-``--directory-filter-types=<video,audio,image>``
+``--directory-filter-types=<video,audio,image,archive,playlist>``
     Media file types to filter when opening directory. If the list is empty,
-    all files are added to the playlist. (Default: ``video,audio,image``)
+    all files are added to the playlist. (Default:
+    ``video,audio,image,archive,playlist``)
```

Comment:
Do we want to open/add archives by default?

Reply:
Well the user of https://github.com/mpv-player/mpv/issues/15096 got screwed by mpv no longer opening archives in directories.

Reply:
Ok. We don't open any other junk now too, but at least we have a chance to open archives.